src/features/import/utils/problems/predictProblems.spec.ts

```diff
@@ -520,4 +520,25 @@ describe('predictProblem()', () => {
       },
     ]);
   });
+
+  it('Removes weird characters from phone fields', () => {
+    const sheet = makeFullSheet({
+      columns: [
+        {
+          field: 'phone',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        { idField: 'id', kind: ColumnKind.ID_FIELD, selected: true },
+      ],
+      firstRowIsHeaders: false,
+      rows: [
+        {
+          data: ['+46 30777 88 68‬', 10],
```

Comment:
I think it would be great if there was a comment either at the beginning of the test or by the line where the "weird characters" are added saying what they are, like you describe in this PR description!

Reply:
Done